third_party/uv/workspace.bzl

```diff
@@ -0,0 +1,17 @@
+"""Provides the repository macro to import libuv."""
+
+load("//third_party:repo.bzl", "tf_http_archive", "tf_mirror_urls")
+
+def repo():
+    """Imports libuv."""
+
+    UV_VERSION = "v1.38.0"
```

Comment:
Is there a reason we are using v1.38.0 and not the latest version (v1.48.0)?

Reply:
I chose the version used in the tests in Gloo: https://github.com/facebookincubator/gloo/blob/main/.circleci/config.yml#L36

It's likely newer versions will work just as well, happy to update.

Reply:
I see. Agreed on matching their version.